.github/workflows/release.yml

```diff
@@ -50,16 +50,14 @@ jobs:
             PACKAGE=$(echo "$COMMIT_MESSAGE" | grep -o "version of TypeScript project [^ ]*" | cut -d " " -f5)
             LANGUAGE="rust"
           fi
-          # Hardcode for now:
-          PACKAGE="all"
```

Comment:
BTW, I suspect this to be the primary reason :point_up: 

With these variables empty, and with the changes below, no releases should be made, unless there was a commit clearly requesting a new release.